package.json

```diff
@@ -12,6 +12,7 @@
     "start": "babel-node src/server.js",
     "lint": "eslint ./src --fix",
     "test": "cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha --timeout 500000 src/tests/*.js --require @babel/register --exit",
+    "test-dev": "cross-env NODE_ENV=test npm run undo:migration && npm run db:migration && nyc --reporter=html --reporter=text mocha --timeout 500000 src/tests/*.js --require @babel/register --exit",
```

Comment:
Please remove this  `test-dev` script

Reply:
The TTl instructed him to put it there

Reply:
The TTL instructed me to remove it previously

Reply:
And then she instructed me to add it again when she saw the need for it

Reply:
Alright